src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#index"
+      >
+        <img
+          src="./images/logo.png"
+          alt="main-logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li>
+            <a
+              href="#Apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">
              Apple
            </a>
          </li>
```